prometheus_juju_exporter/collector.py

```diff
@@ -69,22 +77,16 @@ async def _connect_controller(self, endpoint, username, password, cacert):
                 endpoint=endpoint, username=username, password=password, cacert=cacert
             )
 
-    async def _get_models(self, endpoint, username, password, cacert):
+    async def _get_models(self) -> Dict[str, str]:
```

Comment:
This method can be removed completely. It's just a one-liner

Reply:
Thanks. I've made changes according to these suggestions.